replay-protected-sighash.md

```diff
@@ -46,9 +46,15 @@ The proposed digest algorithm computes the double SHA256 of the serialization of
 
 Items 1, 4, 7 and 9 have the same meaning as in the original algorithm[[3]](#OP_CHECKSIG).
 
+#### outpoint
+
+An outpoint, as used in this document is two fields.
+1. a hash (txid) of the output's containing transaction. (32-byte hash)
+2. the ouput-index (4-byte little endian)
```

Comment:
I searched everywhere, and I could not find where this is defined. Please share where you found this definition. I ended up looking in the code for this.